src/components/SettingsDrawer.tsx

```diff
@@ -162,6 +178,25 @@ const SettingsDrawer: FC<{ open: boolean; onClose: () => void }> = ({
           <IconToggleButton value="5">5</IconToggleButton>
         </ToggleButtonGroup>
       </Box>
+      <Box sx={{ pl: 2, pr: 2 }}>
+        <Heading gutterBottom>Display Testnets</Heading>
```

Comment:
Maybe put the science badge here for "testnets" to create the icon association?

Reply:
Sure, will do